pkg/liqonet/iptables/iptables_test.go

```diff
@@ -1032,6 +1025,162 @@ var _ = Describe("iptables", func() {
 			})
 		})
 	})
+	Describe("Utilities", func() {
+		var (
+			words        = []string{"word0", "word1", "word2", "word3"}
+			expectedIptr IPTableRule
+		)
+		Context("IPTableRule Parser", func() {
+			It("should parse a string", func() {
+				// Tested string: word0
+				expectedIptr = IPTableRule{words[0]}
+				iptr, err := ParseRule(words[0])
+				Expect(err).ToNot(HaveOccurred())
+				Expect(expectedIptr).To(Equal(iptr))
```

Comment:
Given this is common to all It clauses, it should moved to a JustBeforeEach block.